azure/application-gateway/advanced/variables.tf

```diff
@@ -369,4 +369,119 @@ variable "waf_configuration" {
     })), {})
   })
   default = null
+}
+
+variable "listener_waf_configuration" {
+  description = "Defining the WAF policy per listener"
+  type = map(object({ // Key = policy name
+    associated_listeners = list(string)
+    firewall_mode        = optional(string, "Prevention")
+
+    enable_OWASP           = optional(bool, true)
+    OWASP_rule_set_version = optional(string, "3.2")
```

Comment:
latest version is 3.3.5

Reply:
TF Limitation:

https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/web_application_firewall_policy#version